### PR TITLE
qemu: 8.1.3 -> 8.2.0

### PR DESCRIPTION
**Pull request by alyssais**
## Description of changes

- [ChangeLog](https://wiki.qemu.org/ChangeLog/8.2)
- [Planning](https://wiki.qemu.org/Planning/8.2) (ETA: 2023-12-12)

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by alyssais** [comment]
Test failures on x86_64-linux is a regression — previously they all passed.  On the flipside, we've never had them all pass on aarch64-linux before.

**Comment by alyssais** [comment]
Actually they pass locally for me, so I guess it's just timeouts.

**Comment by alyssais** [comment]
@OfBorg test cage cagebreak cloud-init-hostname cloud-init mtp os-prober phosh sway systemd-initrd-luks-fido2 tinywl virtualbox

**Comment by alyssais** [comment]
@willcohen would you like to check for test regressions on Darwin?

**Comment by willcohen** [comment]
Unable to get this working on m2 darwin at the moment:

```
nixpkgs-review pr 267666 --build-args="--fallback"
$ git -c fetch.prune=false fetch --no-tags --force https://github.com/NixOS/nixpkgs master:refs/nixpkgs-review/0 pull/267666/head:refs/nixpkgs-review/1
$ git worktree add /Users/wcohen/.cache/nixpkgs-review/pr-267666-3/nixpkgs 9cc3b66a15941549ba883e7d85b678c6a7f9f8ae
Preparing worktree (detached HEAD 9cc3b66a1594)
Updating files: 100% (38113/38113), done.
HEAD is now at 9cc3b66a1594 Merge pull request #268843 from xrelkd/update/joshuto
$ git merge --no-commit --no-ff 1da792d98a800c15034c245694470b737bafc41c
Automatic merge went well; stopped before committing as requested
$ nix build --nix-path nixpkgs=/Users/wcohen/.cache/nixpkgs-review/pr-267666-3/nixpkgs nixpkgs-overlays=/var/folders/pg/wl09ypf93rq84r1gc2g8ytsw0000gn/T/tmpw8piaiaf --extra-experimental-features nix-command no-url-literals --no-link --keep-going --no-allow-import-from-derivation -f /Users/wcohen/.cache/nixpkgs-review/pr-267666-3/build.nix --fallback
error: a 'x86_64-linux' with features {} is required to build '/nix/store/mxv9g00nyhn76f2zazv3hvg2z8wbqz0l-mounts.sh.drv', but I am a 'aarch64-darwin' with features {benchmark, big-parallel, nixos-test}

Link to currently reviewing PR:
https://github.com/NixOS/nixpkgs/pull/267666

21 packages marked as broken and skipped:
alpine-make-vm-image cloud-init cloud-init.dist collectd diffoscope diffoscope.dist diffoscope.man librenms perl536Packages.SysVirt perl536Packages.SysVirt.devdoc perl538Packages.SysVirt perl538Packages.SysVirt.devdoc qemu_full qemu_xen qemu_xen-light qemu_xen_4_15 qemu_xen_4_15-light rubyPackages_2_7.ruby-libvirt rubyPackages_3_0.ruby-libvirt virt-manager-qt zpool-auto-expand-partitions

35 packages failed to build:
appvm cloud-utils cloud-utils.guest colima cot cot.dist darwin.linux-builder darwin.linux-builder-x86_64 docker-machine-kvm libvirt libvirt-glib libvirt-glib.dev libvirt-glib.devdoc lima lima-bin mgmt nixops_unstable nixops_unstable.dist out-of-tree python310Packages.cot python310Packages.cot.dist python310Packages.libvirt python310Packages.libvirt.dist python311Packages.libvirt python311Packages.libvirt.dist qemu qemu-utils qemu_kvm qemu_test rubyPackages.ruby-libvirt rubyPackages_3_2.ruby-libvirt rubyPackages_3_3.ruby-libvirt virt-manager virt-manager.dist virt-viewer

error: build log of '/nix/store/19dlg4034dw69qbpgiq9kw5i8m2ffr00-appvm-unstable-2021-12-20.drv^*' is not available
error: build log of '/nix/store/035b8wg698w66pfjy2rkcfmfnpd7c9nc-appvm-unstable-2021-12-20' is not available
error: build log of '/nix/store/9svni9xpf5gsj443xx2fhfqzr8h38q00-cloud-utils-0.32.drv^*' is not available
error: build log of '/nix/store/6vxghfznrga3xh0jklzxsy16n3l93pqd-cloud-utils-0.32' is not available
error: build log of '/nix/store/9svni9xpf5gsj443xx2fhfqzr8h38q00-cloud-utils-0.32.drv^*' is not available
error: build log of '/nix/store/b323zxamr5lg4np18ss1glpy5l6p8p1b-cloud-utils-0.32-guest' is not available
error: build log of '/nix/store/ah9z0lnkhdwwcv3yqbam5zz0kzf1f5lx-colima-0.5.6.drv^*' is not available
error: build log of '/nix/store/pppszx8z9wbdwdrqbk112jxnkrlnwizc-colima-0.5.6' is not available
error: build log of '/nix/store/vgf3mav2bdyxa1b15v3c2fhfxnd9bsnz-python3.11-cot-2.2.1.drv^*' is not available
error: build log of '/nix/store/3hk2jsv3mh7inmwy9qxaqjf039lr72v5-python3.11-cot-2.2.1' is not available
error: build log of '/nix/store/vgf3mav2bdyxa1b15v3c2fhfxnd9bsnz-python3.11-cot-2.2.1.drv^*' is not available
error: build log of '/nix/store/fczvj2z092r087ml2jkz1n1qpkga102x-python3.11-cot-2.2.1-dist' is not available
error: build log of '/nix/store/2qncxbbhjlcrcdmh6y5lwah5ay7vc0s4-create-builder.drv^*' is not available
error: build log of '/nix/store/zf726anqvknxb1s0ilhrgah84qrpc9dz-create-builder' is not available
error: build log of '/nix/store/wpy9ikm8iamkvlhxq4rd0prwysx1zx3y-create-builder.drv^*' is not available
error: build log of '/nix/store/a5zm5s5dyd1aq57f0hlm50wdrwaqy3qv-create-builder' is not available
error: build log of '/nix/store/yimsq3hyf2bvrfa6bgkpd9m4fgxgax7i-docker-machine-kvm-0.10.0.drv^*' is not available
error: build log of '/nix/store/sh6kmm8zbm4ag92qzh89iy5y0byh1z8i-docker-machine-kvm-0.10.0' is not available
error: build log of '/nix/store/lyp9z7a2v8zsnl22mwli898jakwsdrxd-libvirt-9.9.0.drv^*' is not available
error: build log of '/nix/store/lfh5rlzcsbnfwpw4fwvnx4pkhl05cbiy-libvirt-9.9.0' is not available
error: build log of '/nix/store/7rpyklw6pv0cqipamzb2gmd63l16447d-libvirt-glib-4.0.0.drv^*' is not available
error: build log of '/nix/store/vgy0k2c57bglp6n9p1fzg0iscgl73371-libvirt-glib-4.0.0' is not available
error: build log of '/nix/store/7rpyklw6pv0cqipamzb2gmd63l16447d-libvirt-glib-4.0.0.drv^*' is not available
error: build log of '/nix/store/x380rp73yp6vw7ifbwcc8cmfj6jgj28l-libvirt-glib-4.0.0-dev' is not available
error: build log of '/nix/store/7rpyklw6pv0cqipamzb2gmd63l16447d-libvirt-glib-4.0.0.drv^*' is not available
error: build log of '/nix/store/pnp9mzlb6xvqpnhb8w96cag12vsxy6v8-libvirt-glib-4.0.0-devdoc' is not available
error: build log of '/nix/store/hgc8vjgd4jpw6ghwlxypixpdi663b07k-lima-0.18.0.drv^*' is not available
error: build log of '/nix/store/az16r43g90k7p52w2ymw9nz31yq5gfma-lima-0.18.0' is not available
error: build log of '/nix/store/hch49q9x3f5by0h76akv2awpqxhqk946-lima-0.17.2.drv^*' is not available
error: build log of '/nix/store/v2a1vci6hq0nc246gcph6crsy67zc556-lima-0.17.2' is not available
error: build log of '/nix/store/sx695fgqavns1pgw3ix6l7d4jh2bf54k-mgmt-unstable-2022-10-24.drv^*' is not available
error: build log of '/nix/store/skqqzj7n2aw1ksh7rhd868bw3gbm0nag-mgmt-unstable-2022-10-24' is not available
error: build log of '/nix/store/ph1scfc6562l2vsh2lavmjm83nlyix7k-nixops-unstable-2023-10-26.drv^*' is not available
error: build log of '/nix/store/51iwx9875x2x1ji3hnlcf0mcxbdp8qs8-nixops-unstable-2023-10-26' is not available
error: build log of '/nix/store/ph1scfc6562l2vsh2lavmjm83nlyix7k-nixops-unstable-2023-10-26.drv^*' is not available
error: build log of '/nix/store/vxzfma1xqvkbh354ny5yf02x4l9scqah-nixops-unstable-2023-10-26-dist' is not available
error: build log of '/nix/store/53f02kcq5pnjfa1ysf55dhfqhcz09s0l-out-of-tree-2.1.1.drv^*' is not available
error: build log of '/nix/store/8ddzhrzv59h5afk54x3kv3ridid9hw8d-out-of-tree-2.1.1' is not available
error: build log of '/nix/store/mzljgih8mmdgizqlya6gfr9indxgjidp-python3.10-cot-2.2.1.drv^*' is not available
error: build log of '/nix/store/zq0jkabgh59s073yapbk23qah083b45p-python3.10-cot-2.2.1' is not available
error: build log of '/nix/store/mzljgih8mmdgizqlya6gfr9indxgjidp-python3.10-cot-2.2.1.drv^*' is not available
error: build log of '/nix/store/1llpw2dxrrqfnbw17s10l42i2nly5hj8-python3.10-cot-2.2.1-dist' is not available
error: build log of '/nix/store/3pgwqcxxgnmw5hcyqwlr0m3n9ry2wid3-python3.10-libvirt-9.9.0.drv^*' is not available
error: build log of '/nix/store/n1f5z9wdks6cc1b3f6ymfjw8hxvxpsvy-python3.10-libvirt-9.9.0' is not available
error: build log of '/nix/store/3pgwqcxxgnmw5hcyqwlr0m3n9ry2wid3-python3.10-libvirt-9.9.0.drv^*' is not available
error: build log of '/nix/store/3a32hjj7byifr3y103djsrzvawmq82h4-python3.10-libvirt-9.9.0-dist' is not available
error: build log of '/nix/store/h49wjq3b7chn19vxafsikh2np5pkm0fx-python3.11-libvirt-9.9.0.drv^*' is not available
error: build log of '/nix/store/090irqfsdi1c1l3mrmg07lyx7knwfknn-python3.11-libvirt-9.9.0' is not available
error: build log of '/nix/store/h49wjq3b7chn19vxafsikh2np5pkm0fx-python3.11-libvirt-9.9.0.drv^*' is not available
error: build log of '/nix/store/0mddkb8wp7p12gciwnd0i481k0q222zg-python3.11-libvirt-9.9.0-dist' is not available
error: build log of '/nix/store/y3xqbx7yrp9ay5dbb6ypvzid5jza1whw-qemu-8.2.0-rc0.drv^*' is not available
error: build log of '/nix/store/72mdy8pffvk99g6nc8lr74fgd37rvkdd-qemu-8.2.0-rc0' is not available
error: build log of '/nix/store/l81rrr66znc5axyqa9dc096vdq1jx8vy-qemu-utils-8.2.0-rc0.drv^*' is not available
error: build log of '/nix/store/rq51400m1z6d9iyia022knwryhlby42y-qemu-utils-8.2.0-rc0' is not available
error: build log of '/nix/store/jzxpcj72yvjc74idf91cyqxiw5pfl2gj-qemu-host-cpu-only-8.2.0-rc0.drv^*' is not available
error: build log of '/nix/store/x35h3chcx5glxfw9q027qpicssmshmic-qemu-host-cpu-only-8.2.0-rc0' is not available
error: build log of '/nix/store/4yjf1f50klyjjbas75rja3rjv5cbg2mv-qemu-host-cpu-only-for-vm-tests-8.2.0-rc0.drv^*' is not available
error: build log of '/nix/store/16cnam7i9yclckqq405fiz02cfnq3s6h-qemu-host-cpu-only-for-vm-tests-8.2.0-rc0' is not available
error: build log of '/nix/store/rk9dbz9f74f1ksv49d96ssqpixa32106-ruby3.1.4-ruby-libvirt-0.8.0.drv^*' is not available
error: build log of '/nix/store/8ia8qcm2xddnsfkkz3f4bglncc211y1v-ruby3.1.4-ruby-libvirt-0.8.0' is not available
error: build log of '/nix/store/mmly7yq1azv1azx9c4yc7v2d5a75af9p-ruby3.2.2-ruby-libvirt-0.8.0.drv^*' is not available
error: build log of '/nix/store/chnis5fr9f6kd9ax1z68z4v979235i9y-ruby3.2.2-ruby-libvirt-0.8.0' is not available
error: build log of '/nix/store/isgwbc1d82npvibr0300yd9fw2bcsi2r-ruby3.3.0-preview3-ruby-libvirt-0.8.0.drv^*' is not available
error: build log of '/nix/store/0znirf6cxqhrshy4y7q49vyk5rfmxf2p-ruby3.3.0-preview3-ruby-libvirt-0.8.0' is not available
error: build log of '/nix/store/jfi0swhks1pz6n5y6278y0v472lb4cnn-virt-manager-4.1.0.drv^*' is not available
error: build log of '/nix/store/3axqk3lifjdfagdy3nzv3wsmsyjxd8jr-virt-manager-4.1.0' is not available
error: build log of '/nix/store/jfi0swhks1pz6n5y6278y0v472lb4cnn-virt-manager-4.1.0.drv^*' is not available
error: build log of '/nix/store/0yf3bi39q69lcz62bbhf88h7c5ril888-virt-manager-4.1.0-dist' is not available
error: build log of '/nix/store/dnr4bg5khx61r0w8y7qy9mxxzsypf9nm-virt-viewer-11.0.drv^*' is not available
error: build log of '/nix/store/sxkyf0y7y6fq9398kqlfndfa9b3xy7gq-virt-viewer-11.0' is not available
```

**Comment by alyssais** [comment]
That looks like QEMU built, but not its dependents, no?  Can you just `nix-build -A qemu.tests https://github.com/NixOS/nixpkgs/tarball/pull/267666/head`?

**Comment by willcohen** [comment]
Yes, apologies! An example of me not having enough coffee before posting.

Summary here: 
```
Summary of Failures:

355/834 qemu:qtest+qtest-aarch64 / qtest-aarch64/readconfig-test                  TIMEOUT         1500.02s   killed by signal 15 SIGTERM
356/834 qemu:qtest+qtest-alpha / qtest-alpha/readconfig-test                      TIMEOUT         1500.01s   killed by signal 15 SIGTERM
365/834 qemu:qtest+qtest-arm / qtest-arm/readconfig-test                          TIMEOUT         1500.01s   killed by signal 15 SIGTERM
372/834 qemu:qtest+qtest-avr / qtest-avr/readconfig-test                          TIMEOUT         1500.02s   killed by signal 15 SIGTERM
381/834 qemu:qtest+qtest-cris / qtest-cris/readconfig-test                        TIMEOUT         1500.01s   killed by signal 15 SIGTERM
389/834 qemu:qtest+qtest-hppa / qtest-hppa/readconfig-test                        TIMEOUT         1500.01s   killed by signal 15 SIGTERM
421/834 qemu:qtest+qtest-i386 / qtest-i386/readconfig-test                        TIMEOUT         1500.02s   killed by signal 15 SIGTERM
433/834 qemu:qtest+qtest-loongarch64 / qtest-loongarch64/readconfig-test          TIMEOUT         1500.02s   killed by signal 15 SIGTERM
440/834 qemu:qtest+qtest-m68k / qtest-m68k/readconfig-test                        TIMEOUT         1500.02s   killed by signal 15 SIGTERM
446/834 qemu:qtest+qtest-microblaze / qtest-microblaze/readconfig-test            TIMEOUT         1500.02s   killed by signal 15 SIGTERM
447/834 qemu:qtest+qtest-microblazeel / qtest-microblazeel/readconfig-test        TIMEOUT         1500.02s   killed by signal 15 SIGTERM
460/834 qemu:qtest+qtest-mips / qtest-mips/readconfig-test                        TIMEOUT         1500.02s   killed by signal 15 SIGTERM
476/834 qemu:qtest+qtest-mips64 / qtest-mips64/readconfig-test                    TIMEOUT         1500.03s   killed by signal 15 SIGTERM
496/834 qemu:qtest+qtest-mips64el / qtest-mips64el/readconfig-test                TIMEOUT         1500.02s   killed by signal 15 SIGTERM
503/834 qemu:qtest+qtest-mipsel / qtest-mipsel/readconfig-test                    TIMEOUT         1500.02s   killed by signal 15 SIGTERM
517/834 qemu:qtest+qtest-nios2 / qtest-nios2/readconfig-test                      TIMEOUT         1500.02s   killed by signal 15 SIGTERM
526/834 qemu:qtest+qtest-or1k / qtest-or1k/readconfig-test                        TIMEOUT         1500.02s   killed by signal 15 SIGTERM
533/834 qemu:qtest+qtest-ppc / qtest-ppc/readconfig-test                          TIMEOUT         1500.02s   killed by signal 15 SIGTERM
544/834 qemu:qtest+qtest-ppc64 / qtest-ppc64/readconfig-test                      TIMEOUT         1500.02s   killed by signal 15 SIGTERM
546/834 qemu:qtest+qtest-riscv32 / qtest-riscv32/readconfig-test                  TIMEOUT         1500.02s   killed by signal 15 SIGTERM
561/834 qemu:qtest+qtest-riscv64 / qtest-riscv64/readconfig-test                  TIMEOUT         1500.01s   killed by signal 15 SIGTERM
579/834 qemu:qtest+qtest-rx / qtest-rx/readconfig-test                            TIMEOUT         1500.02s   killed by signal 15 SIGTERM
613/834 qemu:qtest+qtest-s390x / qtest-s390x/readconfig-test                      TIMEOUT         1500.02s   killed by signal 15 SIGTERM
624/834 qemu:qtest+qtest-sh4 / qtest-sh4/readconfig-test                          TIMEOUT         1500.02s   killed by signal 15 SIGTERM
828/834 qemu:qtest+qtest-sh4eb / qtest-sh4eb/readconfig-test                      TIMEOUT         1500.02s   killed by signal 15 SIGTERM
829/834 qemu:qtest+qtest-sparc / qtest-sparc/readconfig-test                      TIMEOUT         1500.02s   killed by signal 15 SIGTERM
830/834 qemu:qtest+qtest-sparc64 / qtest-sparc64/readconfig-test                  TIMEOUT         1500.02s   killed by signal 15 SIGTERM
831/834 qemu:qtest+qtest-tricore / qtest-tricore/readconfig-test                  TIMEOUT         1500.02s   killed by signal 15 SIGTERM
832/834 qemu:qtest+qtest-x86_64 / qtest-x86_64/readconfig-test                    TIMEOUT         1500.01s   killed by signal 15 SIGTERM
833/834 qemu:qtest+qtest-xtensa / qtest-xtensa/readconfig-test                    TIMEOUT         1500.02s   killed by signal 15 SIGTERM
834/834 qemu:qtest+qtest-xtensaeb / qtest-xtensaeb/readconfig-test                TIMEOUT         1500.03s   killed by signal 15 SIGTERM

Ok:                 683 
Expected Fail:      0   
Fail:               0   
Unexpected Pass:    0   
Skipped:            120 
Timeout:            31  
```

**Comment by willcohen** [comment]
As a followup without pasting the entire test log, all of these are timing out because they can't connect to SPICE, which if I recall correctly has always been true. LGTM!

File: pkgs/applications/virtualization/qemu/default.nix [hunk 1]
```diff
@@ -53,19 +54,19 @@ stdenv.mkDerivation (finalAttrs: {
     + lib.optionalString hostCpuOnly "-host-cpu-only"
     + lib.optionalString nixosTestRunner "-for-vm-tests"
     + lib.optionalString toolsOnly "-utils";
-  version = "8.1.3";
+  version = "8.2.0-rc3";
```
Comment (matdibu) [inline pkgs/applications/virtualization/qemu/default.nix:57]: `8.2.0-rc4` was released yesterday https://github.com/qemu/qemu/compare/v8.2.0-rc3...v8.2.0-rc4, if it's within the scope of this PR

File: pkgs/applications/virtualization/qemu/default.nix [hunk 2]
```diff
@@ -53,19 +54,19 @@ stdenv.mkDerivation (finalAttrs: {
...
 
   src = fetchurl {
```
Comment (matdibu) [inline pkgs/applications/virtualization/qemu/default.nix:59]: would there be a tangible advantage when using the GitHub mirror at https://github.com/qemu/qemu, and switching to `fetchFromGitHub` ?

Comment (alyssais) [reply pkgs/applications/virtualization/qemu/default.nix:59]: No.  First, building releases is what the release tarballs are for, and second, why would we use GitHub and not the actual upstream repository on GitLab?

File: pkgs/applications/virtualization/qemu/default.nix (hunk 1 above)
Comment (alyssais) [reply pkgs/applications/virtualization/qemu/default.nix]: Indeed it is.  Updated.

File: pkgs/applications/virtualization/qemu/default.nix (hunk 2 above)
Comment (matdibu) [reply pkgs/applications/virtualization/qemu/default.nix:59]: That makes sense, thank you for the additional info!

**Comment by alyssais** [comment]
Assuming it passes tests, I intend to this PR with 8.2.0 after it's released, unless anybody objects by then.

**Comment by matdibu** [comment]
There's a 5h old v8.2.0 tag https://gitlab.com/qemu-project/qemu/-/tags/v8.2.0 🎉

**Review by willcohen — APPROVED** [review]

**Review by philiptaron — APPROVED** [review]
Let's go get that freshness!

**Review by matdibu — APPROVED** [review]

**Review by philiptaron — APPROVED** [review]
Result of `nixpkgs-review pr 267666` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>12 packages marked as broken and skipped:</summary>
  <ul>
    <li>qemu_xen</li>
    <li>qemu_xen-light</li>
    <li>qemu_xen-light.debug</li>
    <li>qemu_xen-light.ga</li>
    <li>qemu_xen.debug</li>
    <li>qemu_xen.ga</li>
    <li>qemu_xen_4_15</li>
    <li>qemu_xen_4_15-light</li>
    <li>qemu_xen_4_15-light.debug</li>
    <li>qemu_xen_4_15-light.ga</li>
    <li>qemu_xen_4_15.debug</li>
    <li>qemu_xen_4_15.ga</li>
  </ul>
</details>
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>linuxPackages_testing_bcachefs.virtualbox (linuxKernel.packages.linux_testing_bcachefs.virtualbox)</li>
  </ul>
</details>
<details>
  <summary>82 packages built:</summary>
  <ul>
    <li>alpine-make-vm-image</li>
    <li>cloud-init</li>
    <li>cloud-init.dist</li>
    <li>cloud-utils</li>
    <li>cloud-utils.guest</li>
    <li>colima</li>
    <li>cot (python311Packages.cot)</li>
    <li>cot.dist (python311Packages.cot.dist)</li>
    <li>diffoscope</li>
    <li>diffoscope.dist</li>
    <li>diffoscope.man</li>
    <li>gnome.gnome-boxes</li>
    <li>guestfs-tools</li>
    <li>incus</li>
    <li>libguestfs</li>
    <li>lima</li>
    <li>lima-bin</li>
    <li>linuxKernel.packages.linux_4_19.virtualbox</li>
    <li>linuxPackages_4_19_hardened.virtualbox (linuxKernel.packages.linux_4_19_hardened.virtualbox)</li>
    <li>linuxKernel.packages.linux_5_10.virtualbox</li>
    <li>linuxPackages_5_10_hardened.virtualbox (linuxKernel.packages.linux_5_10_hardened.virtualbox)</li>
    <li>linuxKernel.packages.linux_5_15.virtualbox</li>
    <li>linuxPackages_5_15_hardened.virtualbox (linuxKernel.packages.linux_5_15_hardened.virtualbox)</li>
    <li>linuxKernel.packages.linux_5_4.virtualbox</li>
    <li>linuxPackages_5_4_hardened.virtualbox (linuxKernel.packages.linux_5_4_hardened.virtualbox)</li>
    <li>linuxPackages.virtualbox (linuxKernel.packages.linux_6_1.virtualbox)</li>
    <li>linuxPackages_hardened.virtualbox (linuxPackages_6_1_hardened.virtualbox)</li>
    <li>linuxKernel.packages.linux_6_5.virtualbox</li>
    <li>linuxPackages_6_5_hardened.virtualbox (linuxKernel.packages.linux_6_5_hardened.virtualbox)</li>
    <li>linuxPackages_latest.virtualbox (linuxKernel.packages.linux_6_6.virtualbox)</li>
    <li>linuxPackages_6_6_hardened.virtualbox (linuxKernel.packages.linux_6_6_hardened.virtualbox)</li>
    <li>linuxPackages_latest-libre.virtualbox (linuxKernel.packages.linux_latest_libre.virtualbox)</li>
    <li>linuxPackages-libre.virtualbox (linuxKernel.packages.linux_libre.virtualbox)</li>
    <li>linuxPackages_lqx.virtualbox (linuxKernel.packages.linux_lqx.virtualbox)</li>
    <li>linuxPackages_xanmod.virtualbox (linuxKernel.packages.linux_xanmod.virtualbox)</li>
    <li>linuxPackages_xanmod_latest.virtualbox (linuxKernel.packages.linux_xanmod_latest.virtualbox ,linuxPackages_xanmod_stable.virtualbox)</li>
    <li>linuxPackages_zen.virtualbox (linuxKernel.packages.linux_zen.virtualbox)</li>
    <li>lxd</li>
    <li>mkosi-full</li>
    <li>mkosi-full.dist</li>
    <li>mkosi-full.man</li>
    <li>multipass</li>
    <li>open-watcom-bin</li>
    <li>open-watcom-bin-unwrapped</li>
    <li>out-of-tree</li>
    <li>python310Packages.cot</li>
    <li>python310Packages.cot.dist</li>
    <li>python310Packages.guestfs</li>
    <li>python310Packages.guestfs.dist</li>
    <li>python311Packages.guestfs</li>
    <li>python311Packages.guestfs.dist</li>
    <li>qemu</li>
    <li>qemu-utils</li>
    <li>qemu-utils.debug</li>
    <li>qemu.debug</li>
    <li>qemu.ga</li>
    <li>qemu_full</li>
    <li>qemu_full.debug</li>
    <li>qemu_full.ga</li>
    <li>qemu_kvm</li>
    <li>qemu_kvm.debug</li>
    <li>qemu_kvm.ga</li>
    <li>qemu_test</li>
    <li>qemu_test.debug</li>
    <li>qemu_test.ga</li>
    <li>qtemu</li>
    <li>quickemu</li>
    <li>solo5</li>
    <li>solo5.debug</li>
    <li>tests.testers.nixosTest-example</li>
    <li>tests.testers.runNixOSTest-example</li>
    <li>tests.trivial-builders.references</li>
    <li>vagrant</li>
    <li>virtualbox</li>
    <li>virtualbox.modsrc</li>
    <li>virtualboxHardened</li>
    <li>virtualboxHardened.modsrc</li>
    <li>virtualboxHeadless</li>
    <li>virtualboxHeadless.modsrc</li>
    <li>virtualboxWithExtpack</li>
    <li>virtualboxWithExtpack.modsrc</li>
    <li>zpool-auto-expand-partitions</li>
  </ul>
</details>

**Comment by alyssais** [comment]
If somebody can confirm this builds on aarch64-darwin before OfBorg gets to it, I'll merge it.

File: pkgs/applications/virtualization/qemu/default.nix [hunk 3]
```diff
@@ -53,19 +54,19 @@ stdenv.mkDerivation (finalAttrs: {
...
+    hash = "sha256-vwDS+hIBDfiwrekzcd71jmMssypr/cX1oP+Oah+xvzI=";
   };
 
   depsBuildBuild = [ buildPackages.stdenv.cc ]
     ++ lib.optionals hexagonSupport [ pkg-config ];
 
   nativeBuildInputs = [
     makeWrapper removeReferencesTo
-    pkg-config flex bison meson ninja
+    pkg-config flex bison dtc meson ninja
```
Comment (matdibu) [inline pkgs/applications/virtualization/qemu/default.nix:69]: is `dtc` a new build requirement for `8.2.0`, or does it enable an optional feature in qemu?

Comment (alyssais) [reply pkgs/applications/virtualization/qemu/default.nix:69]: It's required if the `fdt` meson option is enabled.

**Pull request merged**